features/FxA-83-sign-in-confirmation-email/README.md

```diff
@@ -0,0 +1,294 @@
+# Sign-in confirmation email
+
+* [Problem statement](#problem-statement)
+* [Outcomes](#outcomes)
+* [Hypothesis](#hypothesis)
+* [User stories](#user-stories)
+* [Constraints](#constraints)
+* [Proposed solution](#proposed-solution)
+  * [What happens when a Sync user signs in?](#what-happens-when-a-sync-user-signs-in)
+  * [What happens when an OAuth user signs in?](#what-happens-when-an-oauth-user-signs-in)
+  * [What happens when an unverified client requests `/certificate/sign`?](#what-happens-when-an-unverified-client-requests-certificatesign)
+  * [What happens when an unverified client requests `/account/keys`?](#what-happens-when-an-unverified-client-requests-accountkeys)
+  * [How does this affect users on legacy clients?](#how-does-this-affect-users-on-legacy-clients)
+* [Work breakdown](#work-breakdown)
+  * [fxa-content-server](#fxa-content-server)
+  * [fxa-auth-server](#fxa-auth-server)
+  * [fxa-auth-db-mysql](#fxa-auth-db-mysql)
+  * [fxa-auth-mailer](#fxa-auth-mailer)
+* [Mock-ups](#mock-ups)
+  * [Confirm this sign-in screen](#confirm-this-sign-in-screen)
+  * [Confirmation email](#confirmation-email)
+  * [Sign-in confirmed screen](#sign-in-confirmed-screen)
+    * [On the signed-in device](#on-the-signed-in-device)
+    * [Off the signed-in device](#off-the-signed-in-device)
+
+## Problem statement
+
+It is possible for attackers
+armed with a list of known email/password pairs
+to maliciously access users’ Firefox Accounts.
+In such cases,
+attackers would have full access
+to the user’s Sync data,
+including passwords, history, bookmarks and
+the ability to force-install malicious add-ons.
+
+## Outcomes
+
+If an attack occurs,
+we would like fewer accounts to be compromised
+without negatively impacting
+the number of signed-in users
+or our overall FxA engagement rate.
+
+## Hypothesis
+
+If we introduce an additional email confirmation step
+for Sync sign-ins,
+fewer accounts will be compromised.
+
+We will know this to be true if,
+in the event of an attack,
+the number of successful sign-ins remains constant
+and the count of 104 (unverified account) errors
+from protected endpoints
+increases in line with any wider increase in traffic.
+Further, we can measure that
+the confirmation step is not affecting
+legitimate users
+by monitoring the success rate
+of connecting to Sync
+before and after deployment.
```

Comment:
it'll be `screen.<screen_name>.signin.success` That's because a signin occurs either on the signin page itself, or after `signin_permissions`, and now we'll have another interstitial screen `verify_signin`.